netman/adapters/switches/juniper/base.py

```diff
@@ -421,7 +421,8 @@ def set_interface_state(self, interface_id, state):
             self._push(update)
         except RPCError as e:
             self.logger.info("actual setting error was {}".format(e))
-            raise UnknownInterface(interface_id)
+            if e.message != "statement not found: ":
```

Comment:
Would explain this in the source code directly.  A comment would be appreciated :)